src/service/fhir-server-prime/src/main/resources/application.properties

```diff
@@ -1,2 +1,4 @@
 shinnyDataLakeApiImpGuideProfileUri=https://djq7jdt8kb490.cloudfront.net/1115/StructureDefinition-SHINNYBundleProfile.json
-shinnyDataLakeApiUri=https://
\ No newline at end of file
+shinnyDataLakeApiUri=https://
+fhirJsonFolderPath=/home/vinod/workspaces/github.com/vinodaravind01/1115-hub/src/service/fhir-server-prime/src/main/resources
```

Comment:
@raphaelvp @vinodaravind01 @alan-francis please use Java classpaths to obtain resources via relative paths, not using absolute paths with developer names.